app/components/footer-boxes.js

```diff
@@ -0,0 +1,31 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+  didRender(){
```

Comment:
This looks like the indentation is a little off (we are using 4 spaces for tabs even though ember generates 2 spaces).